lms/templates/learner_dashboard/program_details_view.underscore

```diff
@@ -1,7 +1,7 @@
 <header class="js-program-header program-header full-width-banner"></header>
 <section class="program-details-content">
     <div class="program-heading">
-            <% if (inProgressCount === totalCount) { %>
```

Comment:
What you have here would work in most cases, but I think it's more correct to compare `completedCount === totalCount`. If RabbitMQ is down or slow, or a program certificate hasn't be configured for a given program yet, it's possible for a learner to complete the requirements for a given program (i.e., earn all the course certs) without having yet received a program certificate. We should show this message as soon as the user completes the requirements for the program.